src/EditorFeatures/Core/EditorConfigSettings/Data/Whitespace/WhitespaceSetting.cs

```diff
@@ -3,14 +3,27 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Editor.EditorConfigSettings.Updater;
+using Microsoft.CodeAnalysis.EditorConfigSettings.Data;
+using Microsoft.CodeAnalysis.Formatting.Rules;
 using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.EditorConfigSettings.Data
 {
-    internal abstract class WhitespaceSetting
+    internal abstract class WhitespaceSetting : IEditorConfigSettingInfo
     {
+        private static readonly ImmutableArray<string> _boolValues = ImmutableArray.Create(new string[] { "true", "false" });
```

Comment:
Can you add a todo here to switch this over to the editorconfig value holder (with issue tracking it?)